x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/components/controls.tsx

```diff
@@ -138,7 +139,7 @@ export const Controls: FC<Props> = React.memo(
     const nodeType = selectedNode?.data('type');
 
     const onCreateJobClick = useCallback(async () => {
-      const dataViewId = await getDataViewIdFromName(nodeLabel);
+      const dataViewId = await getDataViewIdFromName(data.dataViews, nodeLabel);
```

Comment:
Same here

Reply:
Updated in [70ab770](https://github.com/elastic/kibana/pull/176788/commits/70ab770cc5e06985dba0dd7b86123eee21f7a72e).